config/routes.rb

```diff
@@ -9,6 +9,10 @@
   resources :orders, only: [:index, :create, :show]
   get "/dashboard", to: "users#show", as: "dashboard"
 
+  namespace :admin do
+    resources :dashboard, only: [:index], as: "user"
```

Comment:
interesting... it's not obvious to me why it needs `as: "user"` here but I think it's fine for now, the helper path seems cleaner.